docs/application/web/guides/data/file-system.md

```diff
@@ -108,149 +107,261 @@ To manage file storages:
+
+   ```
+   var directoryPathAndName = 'downloads/exampleDirectory';
+   var destination = 'documents/copyOfExampleDirectory';
+   function onSuccess() {
+       console.log('success');
+   }
+   var errorCallback = function(error) {
+       console.log(error);
+   }
+   tizen.filesystem.copyDirectory(directoryPathAndName, destination, true, onSuccess, errorCallback);
+   ```
+
+3. To move a file, use the `moveFile()` method. The following example moves the files to the `music/` directory. Since the third parameter is set to `false`, no existing files with the same name in the target directory are overwritten.
 
-   for (var i = 0; i < files.length; i++) {
-       documentsDir.copyTo(files[i].fullPath, 'images/backup/' + files[i].name,
-                           true, onSuccess);
+   ```
+   var filePathAndName = 'downloads/exampleFile.mp3';
+   var destination = 'music';
+   function onSuccess() {
+       console.log('success');
    }
+   var errorCallback = function(error) {
+       console.log(error);
+   }
+   tizen.filesystem.moveFile(filePathAndName, destination, true, onSuccess, errorCallback);
    ```
 
-4. To move a file or directory, use the `moveTo()` method. The following example moves the files to the `images/newFolder/` directory. Since the third parameter is set to `false`, no existing files with the same name in the target directory are overwritten.
+4. To move a directory, use the `moveDirectory()` method. The following example moves the directory to the `camera/` directory. Since the third parameter is set to `true`, existing file or directory with the same name in the target directory is overwritten.
 
    ```
-   var files; /* Assume that this is an array of File objects */
+   var directoryPathAndName = 'documents/exampleDirectory';
+   var destination = 'camera';
+   function onSuccess() {
+       console.log('success');
+   }
+   var errorCallback = function(error) {
+       console.log(error);
+   }
+   tizen.filesystem.moveDirectory(directoryPathAndName, destination, true, onSuccess, errorCallback);
+   ```
 
-   for (var i = 0; i < files.length; i++) {
-       documentsDir.moveTo(files[i].fullPath, 'images/newFolder/' + files[i].name,
-                           false, onSuccess);
+5. To rename either a file or a directory, use the `rename()` method. The following example changes file name to the `newName.txt`.
+
+   ```
+   var filePathAndName = 'documents/exampleFile.txt';
+   var newName = 'newName.txt';
+   function onSuccess() {
+       console.log('success');
    }
+   var errorCallback = function(error) {
+       console.log(error);
+   }
+   tizen.filesystem.rename(filePathAndName, newName, onSuccess, errorCallback);
    ```
 
+
 ## Supported Virtual Roots
 
 The virtual roots form a collection of locations that function as a single virtual device file system. The following table lists the supported virtual roots.
```

Comment:
Under Related Information section:
Please rephrase:

•Dependencies:
◦Tizen 2.4 and later for Mobile
◦Tizen 2.3.1 and later for Wearable
◦Tizen 3.0 and later  for TV